apps/api-gateway/src/api/integrations.py

```
@@ -687,3 +687,48 @@ async def get_sync_logs(
+
+
+# --- merged from connectors.py (P0 platform final trim) ---
+
+_connectors_router = APIRouter(prefix="/connectors", tags=["connectors"])
+
+
+@_connectors_router.get("/health")
+async def connectors_health(
+    category: Optional[str] = Query(None, description="pos | delivery | cdp"),
+) -> Dict[str, Any]:
+    """P4-05：连接器健康（last_sync / error / latency）"""
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)